ports/windows/README.md

```diff
@@ -29,13 +32,37 @@ Install following packages using cygwin's setup.exe:
 
 Build using:
 
+    make
+or
     make CROSS_COMPILE=i686-w64-mingw32-
 
 Or for 64bit:
 
     make CROSS_COMPILE=x86_64-w64-mingw32-
 
 
+Building using MSYS2 / Mingw on Windows
+---------------------------------------
+
+Install MSYS2 64 bit from msys2.org
```

Comment:
Does this have to be 64bit?

Reply:
Probably not. I will have to dig out an old laptop and test 32 bit. I am trying to simplify the experience for Windows users and not add multiple options. Is anyone still using 32 bit Windows as a development host? Maybe I could just add a one liner about MYS32.

Reply:
You could just install the 32bit version on a 64bit windows, no need to get out some old hardware. Probably if you just say 'Install MSYS2' it's ok, or possibly something like 'Install MSYS2 (32bit or 64bit depending on hardware/preference)'.